app/models/cat_api.rb

```diff
@@ -0,0 +1,8 @@
+require 'rest-client'
+
+class CatApi
```

Comment:
Nice that you extract this into its own class 👏🏽 Since this is not an Active Record or something that gets instantiated and acts like a Model, a common practice is to place this classes under `app/services`